# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/rocq/coqtop-version.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- coq-9.0.0/toplevel/coqtop.ml.vanilla	2025-07-06 12:06:07.911000000 +0200
+++ coq-9.0.0/toplevel/coqtop.ml	2025-07-06 12:06:25.060000000 +0200
@@ -17,16 +17,7 @@
 let () = at_exit flush_all
 
 let get_version ~boot =
-  if boot then Coq_config.version else
-  try
-    let env = Boot.Env.init () in
-    let revision = Boot.Env.revision env |> Boot.Path.to_string in
-    let ch = open_in revision in
-    let ver = input_line ch in
-    let rev = input_line ch in
-    let () = close_in ch in
-    Printf.sprintf "%s (%s)" ver rev
-  with Sys_error _ | End_of_file -> Coq_config.version
+  Coq_config.version
 
 let print_header ~boot () =
   Feedback.msg_info (str "Welcome to Rocq " ++ str (get_version ~boot));
